packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -442,6 +452,35 @@ class InspectorService extends InspectorServiceBase {
     );
   }
 
+  Future<void> _addPubRootDirectories(List<String> pubDirectories) {
+    // No need to call this from a breakpoint.
```

Comment:
i don't understand this comment - what does it mean?

Reply:
I have no idea either. It looks like i just copy and pasted it from the original.
I'll delete them :)